### PR TITLE
Factory, Multihop, Pool, Pool_stable, Phoenix: removes the user input for token decimals and adds it as a constant instead

### PR DESCRIPTION
**Pull request by gangov**

### CODE REVIEW

File: packages/phoenix/src/lib.rs [hunk 1]
```diff
@@ -1,3 +1,7 @@
 #![no_std]
 
 pub mod utils;
+
+pub mod constants {
+    pub const LP_TOKEN_DECIMALS: u32 = 10;
+}
```
Comment (ueco-jb) [inline packages/phoenix/src/lib.rs:7]: Remove it.
Add it as a parameter for Factory initialization under liquidity pools params.

Comment (ueco-jb) [inline packages/phoenix/src/lib.rs:7]: Remove it.
Add it as a parameter under factory contract initialization.

File: contracts/factory/src/contract.rs [hunk 2]
```diff
@@ -75,6 +79,10 @@ impl FactoryTrait for Factory {
             panic!("Factory: Initialize: there must be at least one whitelisted account able to create liquidity pools.")
         }
 
+        if lp_token_decimals < 6 {
+            panic!("Factory: Initliaze: trying to create factory with invalid param: lp_token_demicals < 6");
+        }
+
```
Comment (ueco-jb) [inline contracts/factory/src/contract.rs:85]: Why is this for? Remove it.

**Review by ueco-jb — APPROVED** [review]
Remove that panic and you can merge.

File: contracts/factory/src/contract.rs (hunk 2 above)
Comment (gangov) [reply contracts/factory/src/contract.rs:85]: I wanted to make sure we don't set some wrongful number for decimals. It's removed

**Pull request merged**